### PR TITLE
[DOC-140] Analytics Middle Tier space migration

### PR DESCRIPTION
**Pull request by chrisgocode**

### CODE REVIEW

File: docs/reference/9-analytics-middle-tier/contributor-guide/contributions/unit-test-framework.md [hunk 1]
```diff
@@ -0,0 +1,611 @@
...
+architecture for those who would like to know more.
+
+With the recent addition of having test cases execute against both SQL Server
+and Postgres, after downloading the Analytics Middle Tier you should review the
+[Analytics Middle Tier Test project
+README](https://github.com/Ed-Fi-Alliance-OSS/Ed-Fi-Analytics-Middle-Tier/blob/main/src/EdFi.AnalyticsMiddleTier.Tests/readme.md)
+and  [this page in Tech
+Docs](https://edfi.atlassian.net/wiki/spaces/EFTD/pages/24807117) that mentions
+the additional setup and caveats to testing against Postgres in addition to SQL
+Server.
```
Comment (roberthunterjr) [inline docs/reference/9-analytics-middle-tier/contributor-guide/contributions/unit-test-framework.md:32]: This links to a Tech Docs page under Analytics Middle Tied Enhancements: https://edfi.atlassian.net/wiki/x/MIJ6AQ 

which is different from the AMT pages being moved currently: https://edfi.atlassian.net/wiki/x/oghwAQ

Does this portion also need to be moved over?

File: docs/reference/9-analytics-middle-tier/user-guide/collections/assessment-collection/asmt_assessmentfact.md
Comment (roberthunterjr) [inline docs/reference/9-analytics-middle-tier/user-guide/collections/assessment-collection/asmt_assessmentfact.md:1]: This table has a bit of horizontal scrolling. Noting so we can revisit in the future.

File: docs/reference/9-analytics-middle-tier/user-guide/collections/chronic-absenteeism-collection/readme.md [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+## Introduction
+
+> "Students who are chronically absent—meaning they miss at least 15 days of
+> school in a year—are at serious risk of falling behind in school. Yet, for too
+> long, this crisis in our nation's public elementary and secondary schools has
+> not been fully understood."
+
+From the introduction to [CHRONIC ABSENTEEISM IN THE NATION'S SCHOOLS: A hidden
+educational crisis](https://www2.ed.gov/datastory/chronicabsenteeism.html) by
+the U.S. Department of Education.
```
Comment (roberthunterjr) [inline docs/reference/9-analytics-middle-tier/user-guide/collections/chronic-absenteeism-collection/readme.md:12]: The link here is no longer valid.

File: docs/reference/9-analytics-middle-tier/user-guide/collections/core-view-collection/studentinternetaccessdim-view-experimental.md [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+added in Data Standard 3.
+
+⚠️ As with many other views, this query relies on the Student's
+relationship with an EducationOrganization, via
+the `StudentEducationOrganization`  table in the ODS. Thus it is possible that
+the Student's InternetAccess could be recorded for the student's relationship
+with one or more agency, including School, Local Education Agency, Service
+Network, etc. For this view's purpose, only the School relationship is
+considered. Data recorded for any other type of Education Agency will be
+ignored.
```
Comment (roberthunterjr) [inline docs/reference/9-analytics-middle-tier/user-guide/collections/core-view-collection/studentinternetaccessdim-view-experimental.md:37]: Do we think these need any additional admonishments? Or perhaps the warnings can all be combined in one area?

**Comment by roberthunterjr** [comment]
We might need to add an entry point to this in the pages/reference.jsx file.

File: docs/reference/9-analytics-middle-tier/contributor-guide/contributions/unit-test-framework.md (hunk 1 above)
Comment (stephenfuqua) [reply docs/reference/9-analytics-middle-tier/contributor-guide/contributions/unit-test-framework.md:32]: At this time I'm not planning on moving the Technical Designs ("EFTD") space out of Confluence, so this link does not need to change...

Oh, but the content itself is outdated. Let's make a note of this for continued improvement without needing to hold up this PR.

File: docs/reference/9-analytics-middle-tier/user-guide/collections/chronic-absenteeism-collection/readme.md (hunk 2 above)
Comment (stephenfuqua) [reply docs/reference/9-analytics-middle-tier/user-guide/collections/chronic-absenteeism-collection/readme.md:12]: We can look for a new one in the next AMT PR.

File: docs/reference/9-analytics-middle-tier/user-guide/collections/core-view-collection/studentinternetaccessdim-view-experimental.md (hunk 3 above)
Comment (stephenfuqua) [reply docs/reference/9-analytics-middle-tier/user-guide/collections/core-view-collection/studentinternetaccessdim-view-experimental.md:37]: Good questions, but let's leave them be for now.

**Comment by stephenfuqua** [comment]
> We might need to add an entry point to this in the pages/reference.jsx file.

Good point... I had deliberately left it out, because we're de-emphasizing AMT at this time. This content will move to the Ed-Fi Exchange area of the documentation in mid year.

File: docs/reference/9-analytics-middle-tier/user-guide/collections/core-view-collection/studentinternetaccessdim-view-experimental.md [hunk 4]
```diff
@@ -0,0 +1,76 @@
...
+This view is already in the `main` branch and will be included in the Analytics
+Middle Tier version 2.5.0. This is considered experimental functionality with
+its initial release and may be subject to significant changes during the course
+of ongoing work with
+the [https://edfi.atlassian.net/wiki/spaces/SKD/pages/26841349](https://edfi.atlassian.net/wiki/spaces/SKD/pages/26841349).
+Because of the experimental nature, patch releases of the Analytics Middle Tier
+might have _breaking changes_ in this view, so it is not recommended that anyone
+take a hard dependency on this view until further notice.
+
+:::
```
Comment (stephenfuqua) [inline docs/reference/9-analytics-middle-tier/user-guide/collections/core-view-collection/studentinternetaccessdim-view-experimental.md:14]:
```suggestion
```

File: docs/reference/9-analytics-middle-tier/user-guide/collections/core-view-collection/studentinternetaccessdim-view-experimental.md [hunk 5]
```diff
@@ -0,0 +1,76 @@
+# StudentInternetAccessDim View - Experimental
```
Comment (stephenfuqua) [inline docs/reference/9-analytics-middle-tier/user-guide/collections/core-view-collection/studentinternetaccessdim-view-experimental.md:1]:
```suggestion
# StudentInternetAccessDim View
```

File: docs/reference/9-analytics-middle-tier/user-guide/collections/core-view-collection/academictimeperioddim-view-experimental.md [hunk 6]
```diff
@@ -0,0 +1,63 @@
+# AcademicTimePeriodDim View - Experimental
```
Comment (stephenfuqua) [inline docs/reference/9-analytics-middle-tier/user-guide/collections/core-view-collection/academictimeperioddim-view-experimental.md:1]:
```suggestion
# AcademicTimePeriodDim View
```

File: docs/reference/9-analytics-middle-tier/user-guide/collections/core-view-collection/studentinternetaccessdim-view-experimental.md (hunk 3 above)
Comment (stephenfuqua) [reply docs/reference/9-analytics-middle-tier/user-guide/collections/core-view-collection/studentinternetaccessdim-view-experimental.md:37]: Now that I'm making other changes, I'll update these to use an admonishment.

Comment (stephenfuqua) [inline docs/reference/9-analytics-middle-tier/user-guide/collections/core-view-collection/studentinternetaccessdim-view-experimental.md:37]:
```suggestion
As with many other views, this query relies on the Student's
relationship with an EducationOrganization, via
the `StudentEducationOrganization`  table in the ODS. Thus it is possible that
the Student's InternetAccess could be recorded for the student's relationship
with one or more agency, including School, Local Education Agency, Service
Network, etc. For this view's purpose, only the School relationship is
considered. Data recorded for any other type of Education Agency will be
ignored.

:::warning

Not available for Data Standard 2.2, as the underlying entities were
added in Data Standard 3.

:::

```

File: docs/reference/9-analytics-middle-tier/user-guide/collections/core-view-collection/studentinternetaccessdim-view-experimental.md [hunk 7]
```diff
@@ -0,0 +1,76 @@
...
+the `StudentEducationOrganization`  table in the ODS. Thus it is possible that
+the Student's InternetAccess could be recorded for the student's relationship
+with one or more agency, including School, Local Education Agency, Service
+Network, etc. For this view's purpose, only the School relationship is
+considered. Data recorded for any other type of Education Agency will be
+ignored.
+
+For more information on recording appropriate student indicator data for
+Internet Access, please
+see [DIGITAL EQUITY COLLECTION](https://edfi.atlassian.net/wiki/spaces/EFDSDRAFT/pages/22773854).
```
Comment (stephenfuqua) [inline docs/reference/9-analytics-middle-tier/user-guide/collections/core-view-collection/studentinternetaccessdim-view-experimental.md:41]: This link to Confluence is still valid.

File: docs/reference/9-analytics-middle-tier/user-guide/collections/early-warning-system-collection/readme.md [hunk 8]
```diff
@@ -0,0 +1,231 @@
...
+    View](./ews_studentsectiongradefact-view.md)
+
+The data model contains two fact tables with different date
+dimensions: [`StudentSectionGradeFact`](./ews_studentsectiongradefact-view.md) represents
+a grade assigned to a date range (grading period),
+and [`StudentEarlyWarningFact`](./ews_studentearlywarningfact-view.md) brings
+together attendance and behavior information on a daily basis. Also
+see [Dimensional
+Views](https://github.com/Ed-Fi-Alliance/Ed-Fi-X-Analytics-Middle-Tier/blob/develop/docs/dimensional-views.md) for
+more information on the other views in this model.
```
Comment (stephenfuqua) [inline docs/reference/9-analytics-middle-tier/user-guide/collections/early-warning-system-collection/readme.md:31]:
```suggestion
together attendance and behavior information on a daily basis.
```

File: docs/reference/9-analytics-middle-tier/user-guide/collections/early-warning-system-collection/readme.md [hunk 9]
```diff
@@ -0,0 +1,231 @@
...
+## Installation
+
+Install using the option code "EWS"
+
+```powershell
+.\EdFi.AnalyticsMiddleTier.Console.exe -c "..." -o EWS
+```
+
+For more information, see the [AMT Deployment
+Guide](https://edfi.atlassian.net/wiki/display/EDFITOOLS/AMT+Deployment+Guide).
```
Comment (stephenfuqua) [inline docs/reference/9-analytics-middle-tier/user-guide/collections/early-warning-system-collection/readme.md:90]:
```suggestion
Guide](../../../deployment-guide).
```

File: docs/reference/9-analytics-middle-tier/user-guide/collections/early-warning-system-collection/readme.md [hunk 10]
```diff
@@ -0,0 +1,231 @@
...
+| D   | 65.0 |
+| F   | 55.0 |
+
+That is, a grade of A will be treated a 95.0, a grade of B will be treated as
+85.0, and so forth.
+
+## Sample Scripts
+
+The source code repository contains a number of sample scripts in the **[samples
+directory](https://github.com/Ed-Fi-Alliance/Ed-Fi-X-Analytics-Middle-Tier/blob/main/samples)**.
```
Comment (stephenfuqua) [inline docs/reference/9-analytics-middle-tier/user-guide/collections/early-warning-system-collection/readme.md:117]:
```suggestion
directory](https://github.com/Ed-Fi-Alliance-OSS/Ed-Fi-Analytics-Middle-Tier/tree/main/samples)**.
```

File: docs/reference/9-analytics-middle-tier/user-guide/collections/early-warning-system-collection/readme.md [hunk 11]
```diff
@@ -0,0 +1,231 @@
...
+```text
+English Grade := during a grading period, the average of grades earned in sections with Subject of English Language Arts, Reading, or Writing.
+
+Math Grade := during a grading period, the average of grades earned in sections with Subject of Mathematics.
+```
+
+These calculations can be see
+in `StudentGradeByGradingPeriod.sql`, `RiskIndicators.sql`, and EwsViews.sql (in
+the [sample
+scripts](https://github.com/Ed-Fi-Alliance/Ed-Fi-X-Analytics-Middle-Tier/tree/master/samples)).
```
Comment (stephenfuqua) [inline docs/reference/9-analytics-middle-tier/user-guide/collections/early-warning-system-collection/readme.md:204]:
```suggestion
scripts](https://github.com/Ed-Fi-Alliance-OSS/Ed-Fi-Analytics-Middle-Tier/tree/main/samples)).
```

File: docs/reference/9-analytics-middle-tier/user-guide/collections/early-warning-system-collection/readme.md [hunk 12]
```diff
@@ -0,0 +1,231 @@
...
+below 65.0 in either Math or English classes as at risk, and they provide an
+early warning flag for a grade below 72.0.
+
+### Data Mart
+
+As noted in [Limiting Impact on the Production
+ODS](../../../deployment-guide/limiting-impact-on-the-production-ods.md),
+running analytics queries over these views can have a negative impact on the
+overall database performance. A "data mart" solution was developed for the
+[https://edfi.atlassian.net/wiki/spaces/EXCHANGE/pages/22487683](https://edfi.atlassian.net/wiki/spaces/EXCHANGE/pages/22487683),
```
Comment (stephenfuqua) [inline docs/reference/9-analytics-middle-tier/user-guide/collections/early-warning-system-collection/readme.md:220]:
```suggestion
[AWS QuickSight Starter Kit](https://edfi.atlassian.net/wiki/spaces/EXCHANGE/pages/22487683),
```

File: docs/reference/9-analytics-middle-tier/user-guide/collections/educator-preparation-collection/readme.md [hunk 13]
```diff
@@ -0,0 +1,41 @@
...
+Kit](https://edfi.atlassian.net/wiki/spaces/SK/pages/22020300/Program+Diversity+and+Persistence+Starter+Kit)
+
+> _Does the diversity of our candidate pool align with the areas the candidates
+> will most likely be teaching in?_
+>
+> _Is our candidate pipeline (Candidate to Completed to Certified to teach) as
+> healthy as we would expect?_
+
+For the [Clinical Experience and Performance Starter
+Kit](https://edfi.atlassian.net/wiki/spaces/SK/pages/22021402/Clinical+Experience+and+Performance+Starter+Kit)
```
Comment (stephenfuqua) [inline docs/reference/9-analytics-middle-tier/user-guide/collections/educator-preparation-collection/readme.md:18]: These two starter kit links will need to change soon, but they're ok for now.

File: docs/reference/9-analytics-middle-tier/user-guide/collections/educator-preparation-collection/readme.md [hunk 14]
```diff
@@ -0,0 +1,41 @@
...
+## Views in this Collection
+
+* [CandidateDim](https://edfi.atlassian.net/wiki/display/EDFITOOLS/EPP_CandidateDim)
+* [CandidateSurveyDim](https://edfi.atlassian.net/wiki/display/EDFITOOLS/EPP_CandidateSurveyDim)
+* [EPPDim](https://edfi.atlassian.net/wiki/display/EDFITOOLS/EPP_EPPDim)
+* [FinancialAidDim](https://edfi.atlassian.net/wiki/display/EDFITOOLS/EPP_FinancialAidDim)
+* [EvaluationElementRatingDim](https://edfi.atlassian.net/wiki/display/EDFITOOLS/Copy+of+EPP_EvaluationElementRatingDim)
+* [RaceDescriptorDim](https://edfi.atlassian.net/wiki/display/EDFITOOLS/EPP_RaceDescriptorDim)
+* [SexDescriptorDim](https://edfi.atlassian.net/wiki/display/EDFITOOLS/EPP_SexDescriptorDim)
+* [TermDescriptorDim](https://edfi.atlassian.net/wiki/display/EDFITOOLS/EPP_TermDescriptorDim)
```
Comment (stephenfuqua) [inline docs/reference/9-analytics-middle-tier/user-guide/collections/educator-preparation-collection/readme.md:32]:
```suggestion
* [CandidateDim](./epp_candidatedim.md)
* [CandidateSurveyDim](./epp_candidatesurveydim.md)
* [EPPDim](./epp_eppdim.md)
* [FinancialAidDim](./epp_financialaiddim.md)
* [EvaluationElementRatingDim](./epp_evaluationelementratingdim.md)
* [RaceDescriptorDim](./epp_racedescriptordim)
* [SexDescriptorDim](./epp_sexdescriptordim.md)
* [TermDescriptorDim](./epp_termdescriptordim.md)
```

File: docs/reference/9-analytics-middle-tier/user-guide/collections/educator-preparation-collection/readme.md [hunk 15]
```diff
@@ -0,0 +1,41 @@
...
+
+## Installation
+
+Install using the option code "EQUITY"
+
+`.\EdFi.AnalyticsMiddleTier.Console.exe -c "..." -o EPP`
+
+For more information, see the [AMT Deployment
+Guide](https://edfi.atlassian.net/wiki/display/EDFITOOLS/AMT+Deployment+Guide).
```
Comment (stephenfuqua) [inline docs/reference/9-analytics-middle-tier/user-guide/collections/educator-preparation-collection/readme.md:41]:
```suggestion
Guide](../../../deployment-guide/readme.mdx).
```

File: docs/reference/9-analytics-middle-tier/user-guide/collections/engage-online-learners-collection/readme.md [hunk 16]
```diff
@@ -0,0 +1,58 @@
...
+## Installation
+
+Install using the option code "engage"
+
+```powershell
+.\EdFi.AnalyticsMiddleTier.Console.exe -c "..." -o engage
+```
+
+For more information, see the [AMT Deployment
+Guide](https://edfi.atlassian.net/wiki/display/EDFITOOLS/AMT+Deployment+Guide).
```
Comment (stephenfuqua) [inline docs/reference/9-analytics-middle-tier/user-guide/collections/engage-online-learners-collection/readme.md:58]:
```suggestion
Guide](../../../deployment-guide/readme.md).
```

File: docs/reference/9-analytics-middle-tier/user-guide/collections/readme.md [hunk 17]
```diff
@@ -0,0 +1,44 @@
...
+| ​EWS | [Early Warning System​](./early-warning-system-collection/readme.md) | _​Required for the [https://edfi.atlassian.net/wiki/spaces/EXCHANGE/pages/22487840](https://edfi.atlassian.net/wiki/spaces/EXCHANGE/pages/22487840).<br/>_   Note, a separate collection to support the AWS QuickSight EWS Starter Kit can be found [https://edfi.atlassian.net/wiki/spaces/EXCHANGE/pages/22488932](https://edfi.atlassian.net/wiki/spaces/EXCHANGE/pages/22488932). |
+| CHRAB | [Chronic Absenteeism](https://edfi.atlassian.net/wiki/display/EDFITOOLS/Chronic+Absenteeism+Collection) | _Required for the [Chronic Absenteeism Quick Start Guide for Metabase](https://github.com/Ed-Fi-Exchange-OSS/Ed-Fi-MetabaseChronicAbsenteeismQuickStart)<br/>_   Can be used to evaluate chronic absenteeism as described in [CHRONIC ABSENTEEISM IN THE NATION'S SCHOOLS: A hidden educational crisis](https://www2.ed.gov/datastory/chronicabsenteeism.html) by the U.S. Department of Education. |
```
Comment (stephenfuqua) [inline docs/reference/9-analytics-middle-tier/user-guide/collections/readme.md:22]:
```suggestion
| CHRAB | [Chronic Absenteeism](./chronic-absenteeism-collection/readme.md) | _Required for the [Chronic Absenteeism Quick Start Guide for Metabase](https://github.com/Ed-Fi-Exchange-OSS/Ed-Fi-MetabaseChronicAbsenteeismQuickStart)<br/>_   Can be used to evaluate chronic absenteeism as described in [CHRONIC ABSENTEEISM IN THE NATION'S SCHOOLS: A hidden educational crisis](https://www2.ed.gov/datastory/chronicabsenteeism.html) by the U.S. Department of Education. |
```

File: docs/reference/9-analytics-middle-tier/user-guide/collections/readme.md [hunk 18]
```diff
@@ -0,0 +1,44 @@
...
+| ASMT | [Assessment](https://edfi.atlassian.net/wiki/display/EDFITOOLS/Assessment+Collection) | _Required for the Assessment Starter Kit<br/>_   Can be used to see students scores, performance levels, and learning standards across assessments and objective assessments |
```
Comment (stephenfuqua) [inline docs/reference/9-analytics-middle-tier/user-guide/collections/readme.md:23]:
```suggestion
| ASMT | [Assessment](./assessment-collection/readme.md) | _Required for the Assessment Starter Kit<br/>_   Can be used to see students scores, performance levels, and learning standards across assessments and objective assessments |
```

File: docs/reference/9-analytics-middle-tier/user-guide/collections/readme.md [hunk 19]
```diff
@@ -0,0 +1,44 @@
...
+| EQUITY | [Student Equity](https://edfi.atlassian.net/wiki/display/EDFITOOLS/Student+Equity+Collection) | _Required for the Student Equity Starter Kit<br/>_   Includes student discipline, feeder schools, student food service programs, and student history. |
```
Comment (stephenfuqua) [inline docs/reference/9-analytics-middle-tier/user-guide/collections/readme.md:24]:
```suggestion
| EQUITY | [Student Equity](./student-equity-collection.md) | _Required for the Student Equity Starter Kit<br/>_   Includes student discipline, feeder schools, student food service programs, and student history. |
```

File: docs/reference/9-analytics-middle-tier/user-guide/collections/readme.md [hunk 20]
```diff
@@ -0,0 +1,44 @@
...
+| ENGAGE | [Student Engagement](./engage-online-learners-collection/readme.md) | _Required for the Engage Online Learners Starter Kit<br/>_   Includes student assignment and student assignment submission information. |
+| EPP | [Educator Prep Program](https://edfi.atlassian.net/wiki/display/EDFITOOLS/Educator+Preparation+Collection) | _Clinical Experience and Performance Collection <br/>_   Program Diversity and Persistence |
```
Comment (stephenfuqua) [inline docs/reference/9-analytics-middle-tier/user-guide/collections/readme.md:26]:
```suggestion
| EPP | [Educator Prep Program](./educator-preparation-collection/readme.md) | _Clinical Experience and Performance Collection <br/>_   Program Diversity and Persistence |
```

File: docs/reference/9-analytics-middle-tier/user-guide/collections/readme.md [hunk 21]
```diff
@@ -0,0 +1,44 @@
...
+    Collection](./chronic-absenteeism-collection/readme.md)
+* [Early Warning System
+    Collection](./early-warning-system-collection/readme.md)
+* [Engage Online Learners
+    Collection](./engage-online-learners-collection/readme.md)
+* [Student Equity Collection](./student-equity-collection/readme.md)
+* [Installation Journal
+    Tables](./installation-journal-tables.md)
+* [Educator Preparation
+    Collection](./educator-preparation-collection/readme.md)
```
Comment (stephenfuqua) [inline docs/reference/9-analytics-middle-tier/user-guide/collections/readme.md:44]:
```suggestion
```

File: docs/reference/9-analytics-middle-tier/user-guide/collections/row-level-security-collection/rls_userauthorization-view.md [hunk 22]
```diff
@@ -0,0 +1,43 @@
...
+:::
+
+The `StaffSectionOrganizationAssignmentAssociation`  table is joined to
+the `StaffClassificationDescriptorScope`  to determine if a staff member should
+have "teacher", "principal", or "superintendent" type access.
+
+| Column Name | Source Table | Source Column | Data Type |
+| --- | --- | --- | --- |
+| UserKey | Staff | StaffUniqueId | int |
+| UserScope | analytics.AuthorizationScope via<br/><br/>[StaffClassificationDescriptorRole](https://edfi.atlassian.net/wiki/display/EDFIODS/Staff+Classification+to+Scope+Mapping+Table) | ScopeName | nvarchar(50) |
```
Comment (stephenfuqua) [inline docs/reference/9-analytics-middle-tier/user-guide/collections/row-level-security-collection/rls_userauthorization-view.md:31]:
```suggestion
| UserScope | analytics.AuthorizationScope via<br/><br/>StaffClassificationDescriptorRole | ScopeName | nvarchar(50) |
```

File: docs/reference/9-analytics-middle-tier/whats-new.md [hunk 23]
```diff
@@ -0,0 +1,462 @@
...
+Some of the enhancements were in response to community-identified issues, which
+were never logged as bugs per se. They are listed below for completeness along
+with one community-reported bug.
+
+|     |     |     |     |
+| --- | --- | --- | --- |
+| Key | Summary | T   | Status |
+
+[No issues
+found](https://tracker.ed-fi.org/issues/?jql=key+in+%28BIA-150%2C+BIA-152%2C+BIA-227%2C+BIA-309%29+&src=confmacro)
```
Comment (stephenfuqua) [inline docs/reference/9-analytics-middle-tier/whats-new.md:307]:
```suggestion
were never logged as bugs per se. 
```

File: docs/reference/9-analytics-middle-tier/whats-new.md [hunk 24]
```diff
@@ -0,0 +1,462 @@
...
+
+### Bug Fixes
+
+|     |     |     |
+| --- | --- | --- |
+| Key | Summary | T   |
+
+[No issues
+found](https://tracker.ed-fi.org/issues/?jql=fixVersion+%3D+%22Analytics+Middle+Tier+v1.3.2%22+&src=confmacro)
+
```
Comment (stephenfuqua) [inline docs/reference/9-analytics-middle-tier/whats-new.md:357]:
```suggestion
* StaffClassificationDescriptorScope in Wrong Schema
* ContactPerson Bad Join with ParentAddress in Data Standard 3.1
* Prevent GradingPeriodDimension From Showing Future Grading Periods (v1.3.2)
* Prevent Student Dimension Duplicates Due to StudentSchoolFoodServiceProgramAssociation (v1.3.2)
```

File: docs/reference/9-analytics-middle-tier/whats-new.md [hunk 25]
```diff
@@ -0,0 +1,462 @@
...
+<summary>Click here to expand...</summary>
+
+### Bug Fixes
+
+|     |     |     |
+| --- | --- | --- |
+| Key | Summary | T   |
+
+[No issues
+found](https://tracker.ed-fi.org/issues/?jql=fixVersion+%3D+%22Analytics+Middle+Tier+v1.3.1%22++&src=confmacro)
```
Comment (stephenfuqua) [inline docs/reference/9-analytics-middle-tier/whats-new.md:372]:
```suggestion
* ContactPersonDimension has incorrect joins - production hotfix
```

File: docs/reference/9-analytics-middle-tier/whats-new.md [hunk 26]
```diff
@@ -0,0 +1,462 @@
...
+Student enrollment now accepts students with future exit date as being enrolled,
+whereas before the views were erroneously excluding all students with an
+ExitDate regardless of when that date occurs.
+
+|     |     |     |
+| --- | --- | --- |
+| Key | Summary | T   |
+
+[No issues
+found](https://tracker.ed-fi.org/issues/?jql=issuetype+%3D+Bug+AND+fixVersion+%3D+%22Analytics+Middle+Tier+v1.3.0%22+++&src=confmacro)
```
Comment (stephenfuqua) [inline docs/reference/9-analytics-middle-tier/whats-new.md:410]:
```suggestion
```

File: docs/reference/9-analytics-middle-tier/user-guide/collections/early-warning-system-collection/readme.md [hunk 27]
```diff
@@ -0,0 +1,228 @@
...
+| F   | 55.0 |
+
+That is, a grade of A will be treated a 95.0, a grade of B will be treated as
+85.0, and so forth.
+
+## Sample Scripts
+
+The source code repository contains a number of sample scripts in the **[samples
+directory](https://github.com/Ed-Fi-Alliance-OSS/Ed-Fi-Analytics-Middle-Tier/tree/main/samples)**.
+
```
Comment (roberthunterjr) [inline docs/reference/9-analytics-middle-tier/user-guide/collections/early-warning-system-collection/readme.md:115]: This directory is not valid. I tried searching and found this. https://github.com/Ed-Fi-Alliance/Ed-Fi-Analytics-Middle-Tier/tree/develop/samples

File: docs/reference/9-analytics-middle-tier/amt-overview.md [hunk 28]
```diff
@@ -0,0 +1,229 @@
...
+
+The sections below provide the historical context and functional analysis that
+led to the initial development of the AMT in 2018.
+
+:::
+
+## Background
+
+### Problem Definition
+
```
Comment (roberthunterjr) [inline docs/reference/9-analytics-middle-tier/amt-overview.md:23]: The Background heading here is empty and the same level as the Problem Definition tag below. I'm think making Background level 1 is the easiest adjustment to make it consistent.

**Review by roberthunterjr — CHANGES_REQUESTED** [review]

File: docs/reference/9-analytics-middle-tier/amt-overview.md (hunk 28 above)
Comment (stephenfuqua) [reply docs/reference/9-analytics-middle-tier/amt-overview.md:23]: Problem definition is now at third level. Can only have one first level per document. This now has the expected table of contents.

image

File: docs/reference/9-analytics-middle-tier/user-guide/collections/core-view-collection/gradingperioddim-view/readme.md [hunk 29]
```diff
@@ -0,0 +1,36 @@
...
+
+Analytics end-users may want to slice and filter presentation views by grading
+period information, especially for viewing and aggregating grades. These
+calendar-related dimensions are distinct from the [DateDim
+View](../datedim-view.md) because that view provides an
+industry-standard concept of a Date Table, whereas the school calendar dimension
+fields are specific to the education domain. The two could be combined; however,
+keeping them separate helps isolate their roles and provide flexibility on
+further refinements. Part of the [Core View
+Collection](./readme.md).
```
Comment (roberthunterjr) [inline docs/reference/9-analytics-middle-tier/user-guide/collections/core-view-collection/gradingperioddim-view/readme.md:13]:
```suggestion
Collection](../readme.md).
```

File: docs/reference/9-analytics-middle-tier/user-guide/collections/early-warning-system-collection/readme.md (hunk 27 above)
Comment (stephenfuqua) [reply docs/reference/9-analytics-middle-tier/user-guide/collections/early-warning-system-collection/readme.md:115]: Your link is to the pre-open source version. I'm not having any problems with this link.

File: docs/reference/9-analytics-middle-tier/amt-overview.md (hunk 28 above)
Comment (roberthunterjr) [reply docs/reference/9-analytics-middle-tier/amt-overview.md:23]: I see this was from the old version I had checked out

**Review by roberthunterjr — APPROVED** [review]

**Pull request merged**